packages/gatsby/src/services/calculate-dirty-queries.ts

```diff
@@ -10,7 +10,7 @@ export async function calculateDirtyQueries({
   if (!store) {
     reporter.panic(`Cannot run service without a redux store`)
   }
-  const state = store.getState()
+  const state = store!.getState()
```

Comment:
If `panic` is typed with a return value of `never`, then TypeScript can infer that this will never be reached, so you won't need this assetion.